### ISSUE TITLE
at -- amd repository for SLE12SP1+ doesn't exist ..

### ISSUE THREAD

**Issue by mimi1vx** [openSUSE/repose#35]
Last working repo is for sle12, 

```
$ curl http://www2.ati.com/suse/sle12sp1/repodata/repomd.xml
File not found."% 
```

**Comment by plusky** [comment]
Package should be same fo SLE-12-SP0 and SP1 as there is compactible kernel, apart from this driver is being rebuild upon installation of .rpm thus it should be agnostic of kernel version.

**Comment by plusky** [comment]
Resolved via #58
